README.md

```diff
@@ -1,77 +1,30 @@
-# 1Password Connect
+<p align="center">
+  <img src="https://github.com/1Password/connect/assets/46452606/0f7cf2a8-a290-41fc-b78d-3dfb1017f9be" alt="Header" >
+</p>
```

Comment:
This image was made specifically for Shell Plugins. Working on the `connect-sdk-js` repo today I found the [Introducing 1Password Secrets Automation](https://blog.1password.com/introducing-secrets-automation/) blog post, and I'm using the banner image from that one - IMO we should use it for all of our Connect-related repos.

Code sample from my PR is below. Note that we can also get rid of the enclosing `<p>` tags (not needed) and the `align="center"` attribute (according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#attributes), this attribute is obsolete and shouldn't be used).

```html
<!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
<img alt="" role="img" src="assets/connect-illustration.svg"/>
```